tests/routes/api/v1/metrics_test.py

```diff
@@ -15,31 +15,40 @@ def test_get_metrics(app, minimal_database):
         response = metrics_api()
         assert response == '[]'
 
-    MeetingSubscription(title='test1').put()
+    new_subscription = MeetingSubscription(title='test1')
+    new_subscription.put()
```

Comment:
Not a blocker, but just had this thought: is there a way we can make sure tests clean up after themselves easily?